app/models/bookmark.rb

```diff
@@ -0,0 +1,5 @@
+class Bookmark < ActiveRecord::Base
+  belongs_to :bookmark_group
+
+  validates_presence_of :name, :href, :bookmark_groups_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.